Project/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
+jdk:
+  - oraclejdk8
+deploy:
```

Comment:
Shouldn't there be a test before the deploy?

Reply:
?